install_config/aggregate_logging.adoc

```diff
@@ -967,6 +967,12 @@ $ oc patch secrets/logging-fluentd --type=json \
   --patch "[{'op':'add','path':'/data/your_private_key','value':'$(base64 /path/to/your_private_key.pem)'}]"
 ----
 
+[NOTE]
+====
+Replace `your_private_key` with a generic name. This is not a path on your
+host system, but it is the link to the JSON path.
+====
```

Comment:
@bmcelvee Would this work better as _This is a link to the JSON path, not a path on your host system_?
Do we need to define _JSON path_?